### ISSUE TITLE
[ios] 1.1.1  playback-queue-ended No longer includes track or position.

### ISSUE THREAD

**Issue by curiousdustin** [react-native-kit/react-native-track-player#406]
It is useful to know what track ended when the queue exhausted.

Track and position used to always come through. It seems the normal case no longer includes this data.

Can you please add this back in?

_**From RNTrackPlayer.swift:**_
```
    public func audioPlayer(itemPlaybackEndedWithReason reason: PlaybackEndedReason) {
        if reason == .playedUntilEnd && player.nextItems.count == 0 {
            sendEvent(withName: "playback-queue-ended", body: nil)
        } else if reason == .playedUntilEnd {
            sendEvent(withName: "playback-track-changed", body: [
                "track": (player.currentItem as? Track)?.id,
                "position": player.currentTime,
                "nextTrack": (player.nextItems.first as? Track)?.id,
            ])
        }
    }
```

**Comment by minhtc** [comment]
added data : https://github.com/HomieTech/react-native-track-player/commit/814c3a75e0f34d03edac7a73af5faed6180a35e9

**Comment by dcvz** [comment]
This has been resolved in 4c0483332779995e98e5f9164ad3c7d548f46615 and will be in a new version on npm soon
